src/Google/GoogleHelper.php

```diff
@@ -1327,6 +1327,32 @@ public function find_subdivision_id_by_code( string $code, string $country_code
 		return self::COUNTRY_SUBDIVISIONS[ $country_code ][ $code ]['id'] ?? null;
 	}
 
+	/**
+	 * Gets the list of supported Merchant Center countries from a continent.
+	 *
+	 * @param string $continent_code
+	 *
+	 * @return string[] Returns an array of country codes with each country code used both as the key and value.
+	 *                  For example: [ 'US' => 'US', 'DE' => 'DE' ].
+	 *
+	 * @since x.x.x
+	 */
+	public function get_supported_countries_from_continent( string $continent_code ): array {
+		$countries  = [];
+		$continents = $this->wc->get_wc_countries()->get_continents();
```

Comment:
💅 
There is a method `get_continents` in `src/Proxies/WC.php`, I think we can just use that so we don't need to call `get_continents` method from the instance of `WC_Countries`.

https://github.com/woocommerce/google-listings-and-ads/blob/efe2f25e69c5fc740e433212bfca3880bf5286ef/src/Proxies/WC.php#L75